test/test-remote/looker/src/index.ts

```diff
@@ -177,10 +177,27 @@ async function createNewLogSeriess(
 
     let stream: MetricSeries | LogSeries;
     if (CFG.metrics_mode) {
+      // Calculate amount to subtract from start time, or avoid random() if subtraction is disabled
+      const subtractSecs = (CFG.metrics_past_start_range_max_seconds === 0)
```

Comment:
Thinking this through, I don't think we should allow `metrics_past_start_range_max_seconds` to be 0. If code does not crash, then this should result in _immediate_ throttle. Which is a little weird. TODO: think this through.

Reply:
I think we may also want to rename this params after all:

`--metrics-start-time-delay-lower-bound` 

`--metrics-start-time-delay-upper-bound` 

A difficulty with `max` and `min` in this case are the various corners one has to think around. "max delay" is of course furthest in the past, i.e. the left/lower bound of the time interval. Whereas the way you've used "max" here defines th _minimal_ shift compared to 'now' (the right/upper bound of the time interval).

Reply:
I remember trying this with zero, just to see what would happen, and it indeed gave immediate throttling due to the 10 minute check. At the time I was thinking of making the 10 minute value configurable but didn't get around to it. Maybe it could be somehow autoscaled based on the `maxTimeLeapComparedToPreviousFragmentSeconds` math?

Reply:
> Maybe it could be somehow autoscaled based on ...

Yes. In my current work the interval discussed in this thread here is chosen automatically, based on the wall time coupling parameters (min lag, max lag).